packages/sado-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -25,24 +26,12 @@ export function SelectWalletModal({
 
   const onConnectUnisatWallet = async () => {
     try {
-      if (!(window as any).unisat) {
-        window.open(UNISAT_WALLET_CHROME_EXTENSION_URL);
-        throw Error("UniSat browser extension is not installed");
-      }
-
-      await (window as any).ordit.sdk.wallet.get(
-        {
-          connect: "unisat",
-        },
-        (resp: GetWalletModel) => {
-          console.log(resp);
-          updateAddress(resp.data.addresses[0].address);
-          closeModal();
-        }
-      );
+      await ordit.unisat.getAddresses("testnet");
```

Comment:
how about doing this? 
```
const addresses = await ordit.unisat.getAddresses("testnet");
updateAddress(addresses[0].address);
closeModal();
```

Reply:
Yes I made some changes to this file, I was using the `finally` statement incorrectly.